sdk/formrecognizer/Azure.AI.FormRecognizer/src/DocumentTableCell.cs

```diff
@@ -8,5 +8,32 @@ namespace Azure.AI.FormRecognizer.DocumentAnalysis
     [CodeGenModel("DocumentTableCell")]
     public partial class DocumentTableCell
     {
+        private const int DefaultSpanValue = 1;
+
+        private static readonly DocumentTableCellKind DefaultTableCellKind = DocumentTableCellKind.Content;
+
+        /// <summary>
+        /// Table cell kind.
+        /// </summary>
+        public DocumentTableCellKind Kind => KindPrivate ?? DefaultTableCellKind;
+
+        /// <summary>
+        /// Number of rows spanned by this cell.
+        /// </summary>
+        public int RowSpan => RowSpanPrivate ?? DefaultSpanValue;
+
+        /// <summary>
+        /// Number of columns spanned by this cell.
+        /// </summary>
+        public int ColumnSpan => ColumnSpanPrivate ?? DefaultSpanValue;
+
+        [CodeGenMember("Kind")]
+        private DocumentTableCellKind? KindPrivate { get; }
```

Comment:
We renamed the property `Kind` that's populated during deserialization to `KindPrivate`, and kept it hidden from the customers. The new `Kind` property now acts as a proxy that returns the value returned by the service, if any, or sets the default value otherwise.

Reply:
Is the default a legitimate value that the service ever returns?  If so, is it impactful that callers can't determine whether it was returned or a default?

Reply:
It's not clear to me if there's any scenario where the service actually returns the default value, but it doesn't matter. Taking `Kind` as an example, "not-returned" means "content", "content" means "content". There should be no differentiation from the caller's perspective.

Ideally the generated code should already implement this behavior for us, but we have a possible bug in CodeGen + missing info in the swagger, so we need to do it manually for now.

Reply:
Fair enough; that addresses my concern.